### PR TITLE
Updated planning for Meta-proteomics workflow

### PR DESCRIPTION
**Pull request by anubhav0fnu**
@scanon, the Meta-proteomics workflow.

### CODE REVIEW

File: Task-2.1.5.1.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+### Background  


+The NMDC initiative will enable users in the environmental microbiology community to easily search and access the microbiome (FICUS) data collected within three national laboratories (PNNL, LANL, and LBNL) using a new framework. Currently, the instrument data (e.g. LC-MS/MS analyses), and data analysis results are being managed via independent systems across labs. This framework will make the different datasets and workflows from each sub-area within environmental microbiology available under one canopy.
+
+### Positioning
+
+

The Integrative Omics group at PNNL currently has their own internal data repository (DMS) that contains metadata, results/data files, etc. related to each sample (single organism or group of meta-organisms) analyzed via mass spectrometry for protein identification and quantification. For long term storage and future external accessibility, DMS data is pushed into the MyEMSL data repository. MyEMSL supports different initiatives across multiple divisions at PNNL to achieve and maintain their respective data and metadata sources.
```
Comment (scanon) [inline Task-2.1.5.1.md]: I'm not sure if this is totally relevant.

File: Task-2.1.5.1.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    pymssql                   2.1.4                    
+    requests                  2.22.0           
+    scipy                     1.3.1             
+    seaborn                   0.10.0                    
+    setuptools                41.6.0                    
+
+### Third party software used/other dependencies:
+
+    
(1)    MS-GF+: Universal Database Search Tool for Mass Spectrometry.
```
Comment (scanon) [inline Task-2.1.5.1.md:32]: Is there a github URL or other place to obtain this?

File: Task-2.1.5.1.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                   Sangtae Kim, Pavel A. Pevzner, Nat Commun. 2014 Oct 31;5:5277. doi: 10.1038/ncomms6277. 


+                   
+                   This software is Copyright © 2012, 2013 The Regents of the University of California. All Rights Reserved.
+                   Permission to copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice, this paragraph and the following three paragraphs appear in all copies.
+ 
+    (2)    MASIC+: (MS/MS Automated Selected Ion Chromatogram generator) a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.
```
Comment (scanon) [inline Task-2.1.5.1.md]: Is there a URL?

File: Task-2.1.5.1.md (hunk 1 above)
Comment (anubhav0fnu) [reply Task-2.1.5.1.md]: ok, so do you want to add/amend what's relevant? or you're just suggesting to remove? or you're suggesting that this is beyond what's needed for Meta-proteomics workflow development?

File: Task-2.1.5.1.md (hunk 2 above)
Comment (anubhav0fnu) [reply Task-2.1.5.1.md:32]: Yes, https://github.com/MSGFPlus/msgfplus

File: Task-2.1.5.1.md (hunk 3 above)
Comment (anubhav0fnu) [reply Task-2.1.5.1.md]: for MASIC: https://github.com/PNNL-Comp-Mass-Spec/MASIC

**Comment by anubhav0fnu** [comment]
@scanon, removed irrelevant content and added URLs.

**Pull request closed without merging**